include/nbl/video/IGPUCommandBuffer.h

```diff
@@ -11,6 +11,10 @@
 #include "nbl/video/IGPUCommandPool.h"
 #include "nbl/video/IQueue.h"
 
+#include "git_info.h"
```

Comment:
you don't need to include `git_info`

Reply:
it's required by `logging_macros.h`

Reply:
@AnastaZIuk is it allowed to be a public header and what include path must it have?

Reply:
i kinda don't like we include it without a folder path or anything (think what could happen if another project had same named include)

Reply:
@AnastaZIuk ?